process/smartContract/backwardsCompatibility.go

```diff
@@ -57,7 +57,7 @@ func (sc *scProcessor) addToDevRewardsV1(address []byte, gasUsed uint64, gasPric
 	consumedFee := core.SafeMul(gasPrice, gasUsed)
 
 	var devRwd *big.Int
-	if sc.flagStakingV2.IsSet() {
+	if sc.enableEpochsHandler.IsStakingV2DelegationFlagEnabled() {
```

Comment:
why do we call the StakingV2 activation IsStakingV2DelegationFlagEnabled (contains Delegation)